x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/waterfall_filter.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiFieldSearch,
+  EuiFilterButton,
+  EuiFilterGroup,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCheckbox,
+  EuiPopover,
+  EuiButtonIcon,
+  EuiSpacer,
+} from '@elastic/eui';
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import useDebounce from 'react-use/lib/useDebounce';
+import { FILTER_REQUESTS_LABEL } from '../../waterfall/components/translations';
+import { MimeType } from './types';
+import { OPEN_FILTERS_POPOVER } from '../../translations';
+
+interface Props {
+  query: string;
+  activeFilters: string[];
+  setActiveFilters: Dispatch<SetStateAction<string[]>>;
+  setQuery: (val: string) => void;
+  onlyHighlighted: boolean;
+  setOlyHighlighted: (val: boolean) => void;
+}
+
+export const MIME_FILTERS = [
+  {
+    label: 'XHR',
```

Comment:
Do we have any of these label strings defined elsewhere for reuse and consistency?

Reply:
I'm thinking it's best to err on the side of caution and translate these labels. We have `XHR` translated further up in this diff already, so it'd also be consistent.